### ISSUE TITLE
Oppsummering - knappen "Vis oppsummering"

### ISSUE THREAD

**Issue by AneMarlene** [Artsdatabanken/Fremmedartsbase2023#52]
Om denne skal fungere som i 2018 (noe jeg antar), så skal den produsere en rapport med en oppsummering av hele vurderingen. Her ser du screen-shot av toppen av en slik rapport slik den så ut i 2018: 

![image](https://user-images.githubusercontent.com/58254444/128352802-1c2582bf-c549-4beb-a85d-f732861a614e.png)

Det enkleste er kanskje å se hvordan den ble produsert i 2018-basen/FAB3?

**Comment by AneMarlene** [comment]
Er jeg på rett spor her, @olgahil ?

**Comment by olgahil** [comment]
Ja, Ane er på sporet. Artseksemplet over er en testart der det ligger lite informasjon (en lavart som ikke er fremmed). Normalt sett ser det slik ut (her har jeg brukt en art som er risikovurdert).

![image](https://user-images.githubusercontent.com/60972488/129033744-8fdece35-278b-4b22-a20b-d4f25831ba11.png)
